fixed-requirements.txt

```diff
@@ -41,7 +41,7 @@ six==1.13.0
 # NOTE: sseclient has various issues which sometimes hang the connection for a long time, etc.
 sseclient-py==1.7
 stevedore==1.30.1
-tooz==1.66.1
+tooz==2.8.0
```

Comment:
I hope CI will signalize that, but the requirements change like that you’ll also need to run ‘make requirements’.
See hint here: https://github.com/StackStorm/st2/blob/master/requirements.txt#L1

Reply:
Thanks, that saves me some time. :) Was just about to find out if it's needed for local builds only or if the new requirements file needs to be pushed as well.

Reply:
`make requirements` failes with 

```
Collecting stevedore==1.30.1
  Using cached stevedore-1.30.1-py2.py3-none-any.whl (42 kB)
ERROR: Could not find a version that satisfies the requirement tooz==2.8.0 (from -r requirements.txt (line 62)) (from versions: 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10, 0.11, 0.12, 0.12.1, 0.13.0, 0.13.1, 0.13.2, 0.13.3, 0.14.0, 0.15.0, 0.16.0, 0.17.0, 1.18.0, 1.19.0, 1.20.0, 1.21.0, 1.22.0, 1.23.0, 1.24.0, 1.25.0, 1.26.0, 1.27.0, 1.28.0, 1.29.0, 1.30.0, 1.31.0, 1.32.0, 1.33.0, 1.34.0, 1.35.0, 1.36.0, 1.37.0, 1.38.0, 1.39.0, 1.40.0, 1.41.0, 1.42.0, 1.43.0, 1.43.1, 1.43.2, 1.44.0, 1.45.0, 1.46.0, 1.47.0, 1.48.0, 1.48.1, 1.48.2, 1.49.0, 1.50.0, 1.51.0, 1.52.0, 1.53.0, 1.54.0, 1.55.0, 1.56.0, 1.56.1, 1.57.0, 1.57.1, 1.57.2, 1.57.3, 1.57.4, 1.58.0, 1.58.1, 1.59.0, 1.60.0, 1.60.1, 1.60.2, 1.61.0, 1.62.0, 1.62.1, 1.63.0, 1.63.1, 1.64.0, 1.64.1, 1.64.2, 1.64.3, 1.65.0, 1.66.0, 1.66.1, 1.66.2, 1.66.3, 1.67.0, 1.67.1, 1.67.2)
ERROR: No matching distribution found for tooz==2.8.0 (from -r requirements.txt (line 62))
Makefile:515: recipe for target 'requirements' failed
make: *** [requirements] Error 1
```

Will try to figure it out a bit in ~30 Minutes.

Reply:
```
ERROR: Cannot install -r requirements.txt (line 18), -r requirements.txt (line 62) and oslo.utils<=3.37.0 and >=3.36.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested oslo.utils<=3.37.0 and >=3.36.2
    python-mistralclient 0.1.2.dev508 depends on oslo.utils>=3.33.0
    tooz 2.8.0 depends on oslo.utils>=4.7.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

Updating to tooz 2.8.0 might require removing the Mistral runner entirely. I'm fine with this, but that might turn into a rabbit hole.

Reply:
Is there an in between version that doesn't change Oslo as much? @winem

Reply:
Nevermind. The Mistral runner was ripped out months ago, this branch just needed to be updated from `master`. Just did that, that should fix a lot of things.